src/components/Video/style.scss

```diff
@@ -107,45 +107,46 @@
 }
 
 .caption {
-  color: @color-white;
+  color: $color-white;
   text-align: center;
   position: absolute;
   z-index: 1;
-  top: 64*@unit;
+  top: 64*$unit;
   left: 0;
   right: 0;
 }
 
 .recordingIndicator {
   text-align: center;
-  margin-bottom: 16*@unit;
+  margin-bottom: 16*$unit;
 
   /* Red circle */
   &::before {
     content: '';
     display: inline-block;
-    width: 8*@unit;
-    height: 8*@unit;
-    margin-right: 5*@unit;
-    background: @color-error;
-    border-radius: 4*@unit;
+    width: 8*$unit;
+    height: 8*$unit;
+    margin-right: 5*$unit;
+    background: $color-error;
+    border-radius: 4*$unit;
     line-height: 120%;
   }
 }
+
 .recordingIndicatorText {
-  font-size: @font-size-x-small;
+  font-size: $font-size-x-small;
   line-height: 1.45;
 }
 
-.fullScreenRoundedIcon() {
+@mixin full-screen-rounded-icon() {
```

Comment:
[Minor] For internal (non reusable outside of this file) mixins I'd recommend to start name with an underscore, so: `_full-screen-rounded-icon`